collection.php

```diff
@@ -0,0 +1,40 @@
+<?php require('database.php'); ?>
+<html>
+<head>
+    <title>Collection</title>
```

Comment:
Also add in the following for when using media queries:
    <meta name="viewport" content="width=device-width, initial-scale=1.0">